controllers/storagesystem_controller.go

```diff
@@ -154,6 +162,11 @@ func (r *StorageSystemReconciler) reconcile(instance *odfv1alpha1.StorageSystem,
 		return ctrl.Result{}, err
 	}
 
+	err = r.setOperatorCondition(instance, logger)
```

Comment:
Can you pls add this func in the subscription controller?

Reply:
no qualms, will do.